presentation/src/main/java/com/sopt/presentation/ui/screen/search/composable/SearchScreen.kt

```diff
@@ -70,6 +87,26 @@ fun SearchScreen(
                     }
                 }
             )
+
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(32.dp),
+                contentAlignment = Alignment.Center
+            ) {
+                when {
+                    searchedHobbyUiState is SearchResultUiState.Success -> {
+                        val hobby = (searchedHobbyUiState as SearchResultUiState.Success).hobby
+                        PrimaryText(text = "너의 취미는 $hobby",)
+                    }
+                    searchedHobbyUiState is SearchResultUiState.NotExistUser -> {
+                        PrimaryText(text = "존재하지 않는 사용자입니다.",)
+                    }
+                    searchedHobbyUiState is SearchResultUiState.Loading -> {
+                        CircularProgressIndicator()
+                    }
+                }
```

Comment:
UiState 가 5개인데 3가지만 처리되어 있는 것 같아요! 나머지도 처리 되면 좋을 것 같습니다! 
```suggestion
                
                when(searchedHobbyUiState) {
                    is SearchResultUiState.Success -> {
                        val hobby = (searchedHobbyUiState as SearchResultUiState.Success).hobby
                        PrimaryText(text = "너의 취미는 $hobby",)
                    }
                    is SearchResultUiState.NotExistUser -> {
                        PrimaryText(text = "존재하지 않는 사용자입니다.",)
                    }
                    is SearchResultUiState.Loading -> {
                        CircularProgressIndicator()
                    }
                    // TODO other
                }
```

Reply:
감사합니다 !